### PR TITLE
{VirtualWan} Mark rtv1 and rtv2 as to be deprecated

### PR DESCRIPTION
**Pull request by nagula-ritvika**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by VeryEarly** [comment]
Hi @nagula-ritvika 

Does this PR depends on api 2020-07? It doesn't look like using new SDK

**Comment by nagula-ritvika** [comment]
@VeryEarly currently im just marking them as deprecated. Is there a way i can deprecate the ones which were already deprecated ? this PR doesnt care about the sdk :)

File: src/Network/Network/Cortex/VirtualHub/NewAzureRmVirtualHubRouteCommand.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Commands.Network
     using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
     using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
-    [CmdletDeprecation(ReplacementCmdletName = "Add-AzVirtualHubRoute")]
+    [CmdletDeprecation(ReplacementCmdletName = "New-AzVHubRoute")]
```
Comment (dedhar) [inline src/Network/Network/Cortex/VirtualHub/NewAzureRmVirtualHubRouteCommand.cs:24]: Why did the command name change here

**Review by dedhar — APPROVED** [review]

**Comment by VeryEarly** [comment]
> @VeryEarly currently im just marking them as deprecated. Is there a way i can deprecate the ones which were already marked deprecating ? this PR doesnt care about the sdk :)

If so, please re-target your PR to master branch since this network-july branch is dedicated for an OOB release rely on api 2020-07

File: src/Network/Network/Cortex/VirtualHub/NewAzureRmVirtualHubRouteCommand.cs (hunk 1 above)
Comment (nagula-ritvika) [reply src/Network/Network/Cortex/VirtualHub/NewAzureRmVirtualHubRouteCommand.cs:24]: initially we marked this deprecated and pointed it to use rtv2, so now replaced it to point to rtv3

**Comment by nagula-ritvika** [comment]
Creating new pr

**Comment by nagula-ritvika** [comment]
https://github.com/Azure/azure-powershell/pull/13427

**Pull request closed without merging**